include/etcpal/cpp/thread.h

```diff
@@ -190,6 +191,8 @@ class Thread
 private:
   std::unique_ptr<etcpal_thread_t> thread_;
   EtcPalThreadParams               params_{ETCPAL_THREAD_PARAMS_INIT_VALUES};
+  static const unsigned            MAX_SHUTDOWN_TIME =500; /* half a second */
```

Comment:
Does this work as `static constexpr`? I think that might be a potential optimization.